db/create_tables.sql

```diff
@@ -21,7 +21,7 @@ CREATE TABLE IF NOT EXISTS `checkpoints` (
   `state_machine_name` VARCHAR(255) NOT NULL,
   `state_machine_instance_id` VARCHAR(255) NOT NULL,
   `state_id` BIGINT(20) NOT NULL,
-  `data` LONGTEXT,
+  `data` BLOB,
```

Comment:
No, that data can be derived out of Events & State & State Machine table anyway, right?